apps/Dockerfile.local

```diff
@@ -0,0 +1,61 @@
+ARG JAVA_VERSION=21
+ARG MAVEN_VERSION=3.9.6
+
+FROM maven:${MAVEN_VERSION}-amazoncorretto-${JAVA_VERSION}
```

Comment:
Would it be possible to hook into the configuration we have for jib instead of creating a separate Dockerfile here?

Reply:
Yeah okay, looks like jib is mostly just used to create an image from the existing build process. I thought it was a bit more flexible. Instead of having one `Dockerfile.local`, would it make sense to have a separate one for each application and docker-compose could orchestrate them? That would make it a little clearer how the different services interact with each other and you could view the log output separately per service.

Reply:
> Yeah okay, looks like jib is mostly just used to create an image from the existing build process. I thought it was a bit more flexible. Instead of having one `Dockerfile.local`, would it make sense to have a separate one for each application and docker-compose could orchestrate them? That would make it a little clearer how the different services interact with each other and you could view the log output separately per service.

Could you elaborate a bit on "application" here referring to? are they just the bfd modules under ./apps/?
if we want to filter out the logging events from each bfd components, we probably have to go into the logged records contents e.g. to grep a json record's certain property...?

For services (e.g. API end points exposed on port 6500), there are only 2 services defined in the docker compose, bfd, and db.